fml/platform/darwin/cf_utils.h

```diff
@@ -11,53 +11,99 @@
 
 namespace fml {
 
+/// RAII-based smart pointer wrapper for CoreFoundation objects.
+///
+/// CFRef takes over ownership of the object it wraps and ensures that CFRetain
+/// and CFRelease are called as appropriate on creation, assignment, and
+/// disposal.
 template <class T>
 class CFRef {
  public:
+  /// Creates a new null CFRef.
   CFRef() : instance_(nullptr) {}
 
+  /// Takes over ownership of `instance`, which is expected to be already
+  /// retained.
+  ///
   // NOLINTNEXTLINE(google-explicit-constructor)
```

Comment:
I know this is just a documentation PR, but for potential follow-up: why is this not `explicit`? It seems like a *really* bad idea to have implicit conversion that silently *transfers ownership* away from the original.

For instance, if someone makes a function that takes a `CFRef`, and I call it with an owned local raw CF* variable, that pointer becomes unowned with absolutely nothing in the local code that would indicate that. I would vote strongly for making this `explicit` and fixing any code that was relying on that.

Reply:
Turns out that there were only a few places in the code that required fixing and they were already touched in this patch. Done.